libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/MatrixCells/MatrixCells.tsx

```diff
@@ -44,10 +45,21 @@ export class MatrixCells extends React.PureComponent<IMatrixCellsProps> {
         ? classNames.matrixRow
         : classNames.matrixCol;
 
+    let totalError = 0;
     let falseCount = 0;
+    let maxMetricValue = 0;
+    let metricName: string;
     this.props.jsonMatrix.matrix.forEach((row: any) => {
```

Comment:
are we able to use strong type here instead of any?

Reply:
it's a bit more complicated, let's defer this to a future follow-up PR since this PR is quite big as it is and this is a tangential issue